test/ansible/molecule/cluster/tasks/selector_test.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: Create the test.example.com/v1alpha1.SelectorTest
+  k8s:
+    state: present
+    definition:
+      apiVersion: test.example.com/v1alpha1
+      kind: SelectorTest
+      metadata:
+        name: selector-test
+        namespace: '{{ namespace }}'
+        labels:
+          testLabel: testValue
+      spec:
+        field: value
+    wait: yes
+    wait_timeout: 300
+    wait_condition:
+      type: Running
+      reason: Successful
+      status: "True"
+  register: selector_test
+- name: Assert sentinel ConfigMap has been created for Molecule Test
+  assert:
+    that: cm.data.hello == 'world'
+  vars:
+    cm: "{{ q('k8s', api_version='v1', kind='ConfigMap', namespace=namespace,
+    resource_name='selector-test').0 }}"
```

Comment:
We should probably have a second test case where we create a SelectorTest sans label and verify it doesn't reconcile